performance/README.md

```diff
@@ -0,0 +1,135 @@
+# neo4j-streams performance test
+
+This part of the project aims to give a standard way to understand the performances of the platform and highlight the configuration results.
+
+
+
+Currently, it's available only the test of neo4j to neo4j configuration, in remote too.
+
+## Setup
+
+The basic unit of work used to test the performances needs a specific cypher part in order to get the timestamp of creation and receiving of the event. The elapsed time is calculated using these timestamps through cypher queries.
+
+### Producer
+
+- neo4j 3.4.7 community edition
+- APOC 3.4.0.3
+- 8 GB heap memory
+- CREATE INDEX ON :Performance(group)
+- default neo4j-streams configuration (without filtering)
+- kafka.batch.size=16384
+
+The  creation query used by the tester:
+
+	UNWIND range(1,{msg}) as ran
+	CREATE (n:Performance)
+	SET n.group = {uuid}, n.creation_time = apoc.date.currentTimestamp()
+	RETURN min(n.creation_time) as creation_time
+### Consumer
+
+- neo4j 3.4.7 community edition
+- APOC 3.4.0.3
+- 2 GB heap memory
+- CREATE INDEX ON :Performance(group)
+- default neo4j-streams configuration
+- kafka.max.poll.records=16384
+
+The acquisition query:
+
+```properties
+streams.sink.topic.neo4j=WITH event.value.payload AS payload, event.value.meta AS meta CALL apoc.do.case( [\
+
+payload.type = 'node' AND meta.operation = 'created', \
+
+'CREATE (x:Performance {received_time: apoc.date.currentTimestamp()}) SET x+=props RETURN count(x)'] \
+
+,'RETURN 0', \
+
+{props:  payload.after.properties}) \
+
+YIELD value RETURN count(value)
+```
+
+
+
+## Run the tests
+
+The tester is written in Python. Check your system for:
+
+- Python 3.6
+- py2neo
+- matplotlib
+
+### config.ini
+
+To run the test you need to edit the *config.ini* file in order to setup the connection data for the *producer* and *consumer* neo4j instances.
+
+The *unit* part is used as default values to run the tests. 
+
+- *repeat* is the number of times each test is executed
+- *node* is the number of nodes created for each test
+
+```ini
+[producer]
```

Comment:
shouldn't this be handled by the docker compose?

Reply:
A config file is useful when someone wants to test a real environment.